doc/ref_cert/README.md

```diff
@@ -2,8 +2,137 @@
+requirements to guarantee a smooth overall integration, the same end user
+actions and a unique test result format (e.g. OPNFV test result database)
+needed by the end users and the test case result verification programs (e.g.
+[OVP](https://www.opnfv.org/verification)). Historically, these rules were
+agreed by RC1 team and have been applied since
+[CNTT Snezka](https://github.com/cntt-n/CNTT/wiki/Snezka). The new RC2 stream
+asks to move them in this common place applicable to all CNTT conformance
+suites.
+
+<a name="ri-rc-toolchaings"></a>
+## CNTT RI and RC toolchains
+
+[OPNFV](https://www.opnfv.org/) has built a complete CI/CD toolchain
+continuously deploying and testing NFVI.
+
+As for all OPNFV installer projects,
+[Jenkins](https://build.opnfv.org/ci/view/cntt/) triggers scenario deployments,
+runs the OPNFV gating test cases and then publishes all
+test results in the
+[centralized test database](https://docs.opnfv.org/en/stable-hunter/_images/OPNFV_testing_working_group.png)
+and all artifacts (reports, logs, etc.) to
+[an S3 compatible storage service](http://artifacts.opnfv.org/).
+
+The CNTT verification and conformance processes will leverage existing OPNFV
+testing knowledge (projects) and experience (history) and then will conform
+to the overall toolchain design already in-place. The RC toolchain only
+requires for the local deployment of the components instead of leveraging
+the common OPNFV centralized services. But the interfaces remain unchanged
+mainly leveraging jenkins jobs, the common test case execution, the test
+result database and the S3 protocol to publish the artifacts. It's worth
+mentioning that dumping all results and logs required by conformance is already
+in place in CIRV (see
+[cntt-latest-zip](https://build.opnfv.org/ci/job/cntt-latest-zip/)) and
+Functest daily jobs (see
+[functest-hunter-zip](https://build.opnfv.org/ci/job/functest-hunter-zip/3/console))
+
+It should be noted that
+[Xtesting CI](https://galaxy.ansible.com/collivier/xtesting) supports both
+centralized and distributed deployment models as described below. It has
+deployed the full toolchain in one small virtual machine to verify ONAP Openlab
+via Functest.
+
+<a name="testing-integration-requirements"></a>
+## Test case integration requirements
+
+To reach all goals (verification, compliance and conformance) expected by
+CNTT, all test cases must be delivered as
+[Docker containers](https://www.docker.com/) and meet the requirements to
+simplify the CI toolchain setups:
+- the common test case execution
+- the unified way to manage all the interactions with the CI/CD components and
+  with third-parties (e.g. dump all test case logs and results for
+  conformance)
+
+For their parts, the Docker containers simply enforce that the test cases are
+delivered with all runtime dependencies. Then it prevents lots of manual
+operations when configuring the server running the test cases and prevent
+conflicts between all test case dependencies.
```

Comment:
```suggestion
conflicts between the test case due to any dependencies.
```

Reply:
@pgoyal01 your proposal seems worse. I was trying to explain two tests could leverage different library versions which could conflict. Let me know if I do update my first proposal.

Reply:
@collivier Sorry. Maybe
```suggestion
conflicts between the test cases due to any dependencies.
```